index.html

```diff
@@ -50,5 +50,196 @@ <h1>Hey, I'm Aidan</h1>
             </div>
         </div>
     </section>
+
+    <!--Middle section-->
+
+    <section>
+        <div class="recent">
+         <p>My Recent Works</p>
+         <img src="./images/Indicator.svg" alt="indicator" class="indicator">
+        </div>
+        <div id="work-grid">
+            <div class="work">
+                <div class="multi-post">
+                    <div class="bottom">
+                         <p>Multi-Post Stories</p>
+                         <p>Gain+Glory</p>
+                    </div>
+                    <div class="language">
+                        <ul class="lang">
+                            <li class="lang-list"><button >Ruby on rails</button></li>
+                            <li class="lang-list"><button >css</button></li>
+                            <li class="lang-list"><button >JavaScript</button></li>
+                            <li class="lang-list"><button >html</button></li>
+                        </ul>
+                    </div>
+                    <div class="see">
+                        <button >See Project</button>
+                    </div>
+                </div>
+             </div>
+             <div class="work">
+                <div class="multi-post">
+                    <div class="bottom">
+                         <p>Multi-Post Stories</p>
+                         <p>Gain+Glory</p>
+                    </div>
+                    <div class="language">
+                        <ul class="lang">
+                            <li class="lang-list"><button >Ruby on rails</button></li>
+                            <li class="lang-list"><button >css</button></li>
+                            <li class="lang-list"><button >JavaScript</button></li>
+                            <li class="lang-list"><button >html</button></li>
```

Comment:
- [ ] Your current technologies list does not have the specified `font` as in the Figma design
> Figma
![image](https://user-images.githubusercontent.com/90556221/176662836-9746850c-da30-4660-a9e4-4e03d03773ad.png)

> Yours
![image](https://user-images.githubusercontent.com/90556221/176663360-8d327501-844c-4e82-aaf0-0797ee8f0070.png)

This might be because you have specified styles for the `li` but the `button` does not inherit those styles. You might want to remove the button completely since the technologies are not supposed to be clickable buttons.

Reply:
Resolved